### PR TITLE
Add GitHub actions for Jupyter notebooks

### PR DESCRIPTION
**Pull request by Rio6**
#15 

Not tested at all since I don't know how to trigger it on Github. Might need to merge it into master for it to get picked up.

### CODE REVIEW

**Comment by Rio6** [comment]
oh it runs when I open the PR

**Comment by Rio6** [comment]
Fixed CI-related errors, but the actual script still fails. @MrMinos if you can look at the newest ci log and see what's up that'd be great. I'm not familiar with what the scripts are supposed to do.

**Comment by MrMinos** [comment]
Are you able to see the CI logs? Just checking if there needs to be a permission related thing.

**Comment by MrMinos** [comment]
It seems like it's checking for the cell output correctly. It is failing as intended.

The current master does not work so it makes sense. If we merge this, and I rebase my branch, we should see the CI succeeding on that branch.

File: scripts/jsbsim_utils.py [hunk 1]
```diff
@@ -132,10 +134,10 @@ def simulate(aircraft, op_0, op_list=None, tf=50, show=False, realtime=False, ve
     if op_list is None:
         op_list = []
         
-    root = Path('.').resolve()
     fdm = jsbsim.FGFDMExec(str(root)) # The path supplied to FGFDMExec is the location of the folders "aircraft", "engines" and "systems"
 
     # load model
+    fdm.set_aircraft_path('Aircraft')
```
Comment (MrMinos) [inline scripts/jsbsim_utils.py:140]: Excellent find! Thank you.

**Comment by Rio6** [comment]
yeah we can do that, though we probably need to temporarily un-protect master to merge it. And yes I can see the logs.

**Review by MrMinos — APPROVED** [review]

**Pull request merged**